index.js

```diff
@@ -101,7 +101,135 @@ module.exports = {
    * @return {boolean} If available true, otherwise false.
    */
   isCipherAvailable: function (cipher) {
-    return this.crypto.getCiphers().some(function (name) { return name === cipher; });
+    var localCiphers = [];
+    if (this.crypto.getCiphers().length > 0) {
+      localCiphers = this.crypto.getCiphers()
```

Comment:
Thanks for the PR @anistark !

I am not 100% comfortable with this approach as it simply ignore the empty results from `getCiphers` and override it with hardcoded values that we are not sure if they are supported, and introduce uncertainty for other users. 

I am actually curious why `getCiphers` return an empty array. This seems to indicate a configuration problem. Could you share more details on the issues?